docs/learn/storage/block-sync.md

```diff
@@ -57,4 +57,8 @@ Chaindata snapshots offer a faster alternative to Full Sync. A snapshot is a com
 
 ## Snap Sync
 
-Currently, Kaia nodes do not support the [Snap Sync](https://geth.ethereum.org/docs/fundamentals/sync-modes) method. However, using a chaindata snapshot provides a comparable advantage in terms of faster initial synchronization.
\ No newline at end of file
+Currently, Kaia nodes do not support the [Snap Sync](https://geth.ethereum.org/docs/fundamentals/sync-modes) method. However, using a chaindata snapshot provides a comparable advantage in terms of faster initial synchronization.
+
+## AWS AMI
```

Comment:
Please move this section above Snap Sync. (Chaindata -> AMI -> Snap Sync)

Reply:
Done, PTAL.